Source/LinearProgressView.swift

```diff
@@ -115,8 +115,10 @@ open class LinearProgressView: UIView {
         super.layoutSubviews()
         
         let maxWidth: CGFloat = max(self.frame.width - barInset * 2, 0) //prevent from becoming negative
-        let calculatedWidth: CGFloat = CGFloat((progress - minimumValue) / (maximumValue - minimumValue)) * maxWidth
-        
+        var calculatedWidth: CGFloat = 0
+        if (progress - minimumValue) != 0 && (maximumValue - minimumValue) != 0 {
```

Comment:
Shouldn't checking `(maximumValue - minimumValue) != 0` be  enough?

Reply:
Yes you right, it will be enough